vendor/bat-native-ads/src/bat/ads/internal/base/strings/string_html_parser_util.cc

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/base/strings/string_html_parser_util.h"
+#include "base/strings/strcat.h"
+#include "third_party/re2/src/re2/re2.h"
+
+namespace {
+const int kPrefixPadding = 2;
+}  // namespace
+
+namespace ads {
+
+std::string ParseHtmlTagAttribute(const std::string& html,
```

Comment:
During our screen-share, it looked like this function fails when parsing non-ASCII characters